# import traceback
# def test():
#     try:
#         raise ValueError('错了')
#     except Exception as e:
#         import pdb
#         pdb.set_trace()
#         print(traceback.format_exc())
#         print(e)
#
# if __name__ == '__main__':
#     test()